### ISSUE TITLE
Unable to get log-likelihod for MASS::polr() model

### ISSUE THREAD

**Issue by torfason** [vincentarelbundock/modelsummary#670]
I'm trying to get log-likelihood statistics for a model estimated using `MASS::polr()`. The statistic is available using `broom::glance()` but does not show up in the `modelsummary`. This is not an issue with `gof_map` because the statistic is shown when the same model is estimated with standard `lm()`. Reprex below:

``` r
library(modelsummary)
packageVersion("modelsummary")
#> [1] '1.4.2.9005'

set.seed(42)
d <- tibble::tibble(
  x = rnorm(100),
  y_int = sample(1:5, 100, replace=TRUE),
  y_ord = ordered(y_int)
)

d
#> # A tibble: 100 × 3
#>          x y_int y_ord
#>      <dbl> <int> <ord>
#>  1  1.37       2 2    
#>  2 -0.565      1 1    
#>  3  0.363      4 4    
#>  4  0.633      3 3    
#>  5  0.404      5 5    
#>  6 -0.106      2 2    
#>  7  1.51       4 4    
#>  8 -0.0947     2 2    
#>  9  2.02       3 3    
#> 10 -0.0627     2 2    
#> # ℹ 90 more rows

lm(y_int ~ x, data = d) |> msummary(output="data.frame")
#>         part        term statistic      (1)
#> 1  estimates (Intercept)  estimate    2.852
#> 2  estimates (Intercept) std.error  (0.148)
#> 3  estimates           x  estimate   -0.054
#> 4  estimates           x std.error  (0.143)
#> 5        gof    Num.Obs.                100
#> 6        gof          R2              0.001
#> 7        gof     R2 Adj.             -0.009
#> 8        gof         AIC              366.1
#> 9        gof         BIC              373.9
#> 10       gof    Log.Lik.           -180.037
#> 11       gof           F              0.141
#> 12       gof        RMSE               1.46

MASS::polr(y_ord ~ x, data = d) |> msummary(output="data.frame")
#>         part     term statistic     (1)
#> 1  estimates      1|2  estimate  -1.050
#> 2  estimates      1|2 std.error (0.228)
#> 3  estimates      2|3  estimate  -0.163
#> 4  estimates      2|3 std.error (0.201)
#> 5  estimates      3|4  estimate   0.445
#> 6  estimates      3|4 std.error (0.205)
#> 7  estimates      4|5  estimate   1.515
#> 8  estimates      4|5 std.error (0.260)
#> 9  estimates        x  estimate  -0.061
#> 10 estimates        x std.error (0.172)
#> 11       gof Num.Obs.               100
#> 12       gof      AIC             328.5
#> 13       gof      BIC             341.5
#> 14       gof     RMSE              3.03

# I WOULD HAVE EXPECTED TO SEE Log.Lik. IN THE ABOVE data.frame
# (especially since it is available through broom::glance())

MASS::polr(y_ord ~ x, data = d) |> broom::glance()
#> # A tibble: 1 × 7
#>     edf logLik   AIC   BIC deviance df.residual  nobs
#>   <int>  <dbl> <dbl> <dbl>    <dbl>       <dbl> <dbl>
#> 1     5  -159.  328.  342.     318.          95   100
```

<sup>Created on 2023-09-23 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by vincentarelbundock** [comment]
I recommend you read:

1. Documentation of the ... argument (see in particular metrics)
2. Documentation section on data extraction mechanisms, broom vs performance
3. Section of the vignette on adding custom goodness of fit statistics

**Comment by torfason** [comment]
Thank you very much for the pointers. I did not realize about the different data extraction mechanisms, thought `broom` was what was used. So it seems that performance provides p values, but broom provides log-likelihood. I tried setting

`options(modelsummary_get = "broom")`

which got me log-likelihood, but at the cost of the stars. But setting the following seems to be getting me both at the same time:

`glance_custom.polr <- broom:::glance.polr`

Kudos to you for wading through this jungle and your documentation of it, sorry that I didn't see it and for the issue tracker noise :-)

So appreciate the package!
